core/trino-main/src/main/java/io/trino/execution/resourcegroups/IndexedPriorityQueue.java

```diff
@@ -188,6 +188,11 @@ public Iterator<E> iterator()
         return transform(queue.iterator(), Entry::getValue);
     }
 
+    public Iterator<Prioritized<E>> iteratorPrioritized()
```

Comment:
If you need this only for the test in this commit, then I suggest dropping it.
If you need it elsewhere, then it should go into that commit.
By itself this method addition looks unnecessary

Reply:
could be vestigial - let me check

Reply:
yeah - no longer needed